ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/private-hosted-zone/provision/cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-provision
+  steps:
+    - ref: ipi-conf
+    - ref: aws-provision-vpc-shared
+    - ref: aws-provision-bastionhost
+    - ref: aws-provision-route53-private-hosted-zone
+    - chain: ipi-conf-aws-route53-private-hosted-zone
```

Comment:
pls see my comment in https://github.com/openshift/release/pull/27656/files#r853926489

Reply:
Done